examples/games/game_menu.rs

```diff
@@ -370,7 +370,7 @@ mod menu {
 
     // This system updates the settings when a new value for a setting is selected, and marks
     // the button as the one currently selected
-    fn setting_button<T: Component + PartialEq + Copy>(
+    fn setting_button<T: Component + Sync + PartialEq + Copy>(
```

Comment:
This is an unfortunate bound to need to add in end-user code.

What happens if it's omitted?

Reply:
The bound on `Res<T>` fails to compile. This honestly only comes up in generic code though.